contrib/julia.appdata.xml

```diff
@@ -28,7 +28,7 @@
  </description>
  <screenshots>
   <screenshot type="default">
-   <image>https://julialang.org/images/julia-gnome.png</image>
+   <image>https://raw.githubusercontent.com/JuliaLang/www.julialang.org/main/_assets/images/julia-gnome.png</image>
```

Comment:
We should not point to GitHub, but to our own CDN

Reply:
> point to our own CDN

Noted.

It was the only same named file able to be located.

https://github.com/JuliaLang/www.julialang.org/pull/1773

Reply:
Sorry, I didn't realize this PR also needed updating with the new URL. Can we do that in a follow-up?

Reply:
anyone can fix

this looks like the CDN url

https://julialang.org/assets/images/julia-gnome.png

contrib/julia.appdata.xml

```diff
    <screenshot type="default">
-    <image>https://raw.githubusercontent.com/JuliaLang/www.julialang.org/main/_assets/images/julia-gnome.png</image>
+    <image>https://julialang.org/assets/images/julia-gnome.png</image>
    </screenshot>
```

Reply:
Fixed in #47471